src/sentry/tasks/store.py

```diff
@@ -66,7 +67,10 @@ def _do_preprocess_event(cache_key, data, start_time, event_id, process_event):
     if should_process(data):
         # save another version of data for some projects to generate
         # preprocessing hash that won't be modified by pipeline
-        hash_cache.set(get_raw_cache_key(project_id, data['event_id']), data)
+        try:
+            hash_cache.set(get_raw_cache_key(project_id, data['event_id']), data)
+        except Exception as e:
+            error_logger.exception(six.text_type(e))
```

Comment:
Personally, I try to keep the log message more descriptive about what happened at a glance (e.g. `Could not save raw event for preprocess hash generation due to error: %s` or something) since the exception message will be logged as part of the exception interface anyway. This is probably a subjective preference though.